### PR TITLE
Improvements to logging cube last updated dates

### PR DESCRIPTION
**Pull request by michaelglenister**
Issue: https://trello.com/c/G32v9fRj/569-api-data-timeliness-as-a-user-i-would-like-to-understand-when-a-dataset-was-last-updated-so-that-i-can-understand-what-should-be

### CODE REVIEW

File: municipal_finance/tests/test_cube_updates.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    def test_aged_creditor_v2(self):
+        update_aged_creditor_facts_v2(
+            self.insert_obj,
+            batch_size=4,
+        )
+        manager = get_manager()
+        with manager.get_engine().connect() as connection:
+            cube = get_cube_with_last_updated(connection, manager, "aged_creditor_v2")
+
+        self.assertEqual(cube["last_updated"][0:10], datetime.today().strftime("%Y-%m-%d"))
```
[the remark this answers is not in the document]
Comment (michaelglenister) [reply municipal_finance/tests/test_cube_updates.py]: Thanks, I changed to test to check the entire timestamp. There is a small risk of failure due to a race condition but this should be very rare

There's some weirdness between the timezone the test environment is using and what the backend is using (2 hours). I just adjusted for this in the test

**Comment by michaelglenister** [comment]
@paulmwatson this also needs a final review/approval

File: municipal_finance/tests/test_cube_updates.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (paulmwatson) [reply municipal_finance/tests/test_cube_updates.py]: Could be test environment is github and UTC/GMT while server is SAST, UTC/GMT+2.

**Review by paulmwatson — APPROVED** [review]

File: municipal_finance/tests/test_cube_updates.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    def test_aged_creditor_v2(self):
+        update_aged_creditor_facts_v2(
+            self.insert_obj,
+            batch_size=4,
+        )
+        manager = get_manager()
+        with manager.get_engine().connect() as connection:
+            cube = get_cube_with_last_updated(connection, manager, "aged_creditor_v2")
+        current_time = datetime.today()-timedelta(hours=2)
+        self.assertEqual(cube["last_updated"], current_time.strftime("%Y-%m-%d, %H:%M"))
```
Comment (jbothma) [inline municipal_finance/tests/test_cube_updates.py:49]: this is not a correct ISO timestamp. It should be `%Y-n&dT%H:%M` - `T` separates time date from the time.

Screenshot_2023-01-04_17-18-28

**Pull request merged**